portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* Shows about text when the "About" button is pressed. */
+function showAbout () {
+    const aboutContainer = document.getElementById('aboutSpace');
+    aboutTest = aboutContainer.getElementsByTagName('p');
+    if (aboutTest.length > 0) {
+        aboutText = aboutContainer.getElementsByTagName('p')[0];
+    }
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Change the text in the aboutSpace element 
+    if (aboutText.className === "about-hide") {
+        aboutText.className = "about-show";
+        aboutContainer.className = "background";
+    }
+    
```

Comment:
remove this line

Reply:
I'm not sure what line you mean here?

Reply:
there's a line between the if block and the else block.

it should just be

if () {
} else {
}